.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Alright, this file needs to be removed.

I've requested multiple times now to resolve this comment and it isn't happening.

I'm going to close this PR for now for that reason.

../Frenck

Reply:
Hi @frenck ,
I understand the frustration there is in these cases but @Sim0cYz not being practical didn't understand, I got added to his fork and fixed it.
Please bear with me and reopen so we can get the logo.
If there is anything else, please write to me so I don't prolong it more than necessary and don't take more of your time.

Thank you again for your help